corehq/apps/hqmedia/urls.py

```diff
@@ -32,7 +31,6 @@
     url(r'^upload/$', BulkUploadMultimediaView.as_view(), name=BulkUploadMultimediaView.urlname),
     url(r'^paths/$', ManageMultimediaPathsView.as_view(), name=ManageMultimediaPathsView.urlname),
     url(r'^paths/download/$', download_multimedia_paths, name='download_multimedia_paths'),
-    url(r'^paths/validate/$', validate_multimedia_paths, name='validate_multimedia_paths'),
```

Comment:
nit: removing a URL/view like this isn't safe. scenario:

1. user opens the page in a background tab that links to path validation
2. we deploy this PR
3. that user goes to that tab, to validate paths. 
4. They are greeted by either a 4xx or 500 error

I don't know enough about this workflow to say how big of a risk it is, so not requiring changes, will let you decide

Reply:
Fair point in general. Not worried about it for this PR because no one is using this feature - I released it in pieces because it was originally on a tight timeline and having it partially released seemed potentially useful.